app/javascript/packs/modal.js

```diff
@@ -0,0 +1,36 @@
+document.addEventListener('DOMContentLoaded', function(){
+  var buttons = document.querySelectorAll(".btn");
+  var closes = document.querySelectorAll(".modal-close");
+  
+  function openModal(event) {
+    var modal = document.querySelector("div[id="+event.currentTarget.id+"]");
+    modal.classList.add('opacity-100');
+    modal.classList.add('z-50');
+  }
+  
+  function closeModal(event) {
+    var modal = document.querySelector("div[id="+event.currentTarget.id+"]");
+    modal.classList.remove('opacity-100');
+    modal.classList.remove('z-50');
+  }
+  
+  buttons.forEach(function (b){
+      b.addEventListener('click',openModal);
+    }
+  );
+  
+  closes.forEach(function(c){
+      c.addEventListener('click',closeModal);
+    }
+  );
+});
+
+window.onclick = function(event) {
```

Comment:
WDYT about attaching an `onClick` just to the "modal-openers" rather than always listening all window-on-clicks?

Reply:
We had it implemented that way, but it showed us errors, so we chose to use them in this way.

Reply:
What error did you have?

Reply:
Would you like to pair on this? 😄 

If so, feel free to reach me on slack or gather 👍